icons/bit.svg

```diff
@@ -1 +1 @@
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Bit</title><path d="M10.98 12.96c0 .599-.17 1.072-.51 1.42-.34.349-.826.523-1.458.523a2.75 2.75 0 0 1-.644-.073 2.557 2.557 0 0 1-.546-.194c-.194-.114-.304-.235-.328-.365s-.036-.259-.036-.388V9.802c0-.146.044-.255.133-.328.09-.073.223-.11.401-.11.178 0 .312.037.4.11.09.073.134.182.134.328v1.554a1.499 1.499 0 0 1 .777-.219c.487 0 .887.159 1.203.474.316.316.474.765.474 1.348zm-2.454.947a.539.539 0 0 0 .231.097c.09.016.174.024.255.024.276 0 .486-.085.632-.255.146-.17.218-.417.218-.74 0-.324-.072-.575-.218-.754-.146-.178-.348-.267-.607-.267a.887.887 0 0 0-.51.17v1.725zm4.47-3.814a.61.61 0 0 1-.182.45.61.61 0 0 1-.45.182.64.64 0 0 1-.473-.182.61.61 0 0 1-.183-.45.6.6 0 0 1 .183-.461.666.666 0 0 1 .473-.17.6.6 0 0 1 .632.632zm-.097 1.482v2.866c0 .146-.045.255-.134.328-.089.073-.223.11-.4.11s-.316-.037-.414-.11c-.097-.073-.145-.182-.145-.328v-2.866c0-.146.048-.255.145-.328.098-.073.235-.11.413-.11s.312.037.401.11c.09.073.134.182.134.328zm3.28 2.332c.145 0 .246.06.303.182.056.122.085.23.085.328a.322.322 0 0 1-.049.158.53.53 0 0 1-.146.158 2.009 2.009 0 0 1-.802.17c-.29 0-.526-.053-.704-.158a1.156 1.156 0 0 1-.413-.4 1.196 1.196 0 0 1-.146-.475 5.776 5.776 0 0 1-.024-.546v-1.19h-.291c-.146 0-.255-.04-.328-.122a.49.49 0 0 1-.11-.34.49.49 0 0 1 .11-.34c.073-.081.182-.122.328-.122h.291v-.704c0-.146.045-.255.134-.328.089-.073.222-.11.4-.11.179 0 .312.037.401.11.09.073.134.182.134.328v.704h.68c.146 0 .255.04.328.122a.49.49 0 0 1 .11.34.49.49 0 0 1-.11.34c-.073.081-.182.122-.328.122h-.68v1.19c0 .08.004.166.012.255.008.09.028.174.06.255a.313.313 0 0 0 .122.11.438.438 0 0 0 .195.036.763.763 0 0 0 .121-.012.763.763 0 0 1 .122-.012.475.475 0 0 1 .194-.05zM0 12.012c0-1.668.316-3.23.947-4.688s1.49-2.729 2.575-3.814S5.88 1.567 7.336.935 10.348-.012 12-.012s3.207.316 4.664.947A12.339 12.339 0 0 1 20.49 3.51a11.76 11.76 0 0 1 2.563 3.814c.631 1.457.947 3.02.947 4.688a11.61 11.61 0 0 1-.947 4.664 11.759 11.759 0 0 1-2.563 3.814 12.339 12.339 0 0 1-3.826 2.575c-1.458.631-3.012.947-4.664.947s-3.207-.316-4.664-.947-2.729-1.49-3.814-2.575C2.437 19.405 1.58 18.134.947 16.676S0 13.664 0 12.012zM12 .862c-1.538 0-2.98.292-4.324.875-1.36.583-2.542 1.38-3.547 2.392S2.333 6.32 1.75 7.664c-.583 1.36-.875 2.81-.875 4.348s.292 2.98.875 4.324a11.197 11.197 0 0 0 2.38 3.547 11.196 11.196 0 0 0 3.547 2.38c1.344.583 2.786.875 4.324.875s2.988-.292 4.348-.875a11.272 11.272 0 0 0 3.523-2.38 11.196 11.196 0 0 0 2.38-3.547c.6-1.344.899-2.785.899-4.324s-.3-2.988-.899-4.348c-.583-1.344-1.377-2.522-2.38-3.534s-2.179-1.81-3.523-2.393A10.918 10.918 0 0 0 12 .862zM8.38 3.413a.496.496 0 0 1 .025.34.442.442 0 0 1-.219.267c-.923.486-1.704 1.025-2.344 1.616S4.624 7.008 4.105 7.98a.443.443 0 0 1-.267.218.423.423 0 0 1-.316-.024.443.443 0 0 1-.218-.267.424.424 0 0 1 .024-.316C3.895 6.522 4.53 5.66 5.235 5.004s1.55-1.243 2.538-1.761a.392.392 0 0 1 .34-.036c.114.04.203.109.268.206z"/></svg>
```

Comment:
I see some [small differences](![image](https://github.com/simple-icons/simple-icons/assets/12817388/62694707-732b-42ac-bd6a-55b702de6981)).
Can you take another look to be sure?

Reply:
Which version are you comparing against? There's a colour variation in the header, but this monochrome version was pulled from the footer (it's a font file)

Reply:
I used the SVG from the font file. I can take another look to be sure.

Reply:
I'll try and re-generate from the font file. Bear with while I re-surface the file 🙃